scene-lib/src/scenelib/annotations/el/LocalLocation.java

```diff
@@ -1,91 +1,219 @@
 package scenelib.annotations.el;
 
+import org.objectweb.asm.Label;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
 import java.util.Objects;
 
+import org.checkerframework.checker.nullness.qual.Nullable;
+
 /**
  * A {@link LocalLocation} holds location information for a local
```

Comment:
Please define "location" here, or the class documentation is a bit circular.  Is it a location in bytecode?